### PR TITLE
in-canvas-for-x width:auto instead of width:inherit

### PR DESCRIPTION
**Pull request by RvWensen**
Issue fix.
Discussion can be found https://github.com/zurb/foundation-sites/issues/11124

### CODE REVIEW

**Comment by ncoden** [comment]
Hi @RvWensen,

Here are some recommentations your future PRs:
* Use the same "format" as others PRs for your PR and commits titles: `feat:`, `fix:`, `docs:`, `refactor:`, `clean:`, `chore:`...
* Provide a list of the changes mades with their reasons (even if you end in repeating yourself with what was said in the related issue).
* If your PR closes an issue, said it explicitely (like "Closes #XXX" or "Fixes #XXX")

**Comment by RvWensen** [comment]
Thanks for the feedback!

File: scss/components/_off-canvas.scss [hunk 1]
```diff
@@ -417,7 +417,7 @@ $breakpoint: small
   height: auto;
   position: static;
   background: inherit;
-  width: inherit;
+  width: auto;
```
Comment (ncoden) [inline scss/components/_off-canvas.scss]: This makes sense for `width: auto`, but @SassNinja was there any reasons for going for `inherit` in the first place for properties that does not inherit their value by default ?

Comment (ncoden) [reply scss/components/_off-canvas.scss]: poke @SassNinja

Comment (ncoden) [reply scss/components/_off-canvas.scss]: Done some research. The incanvas feature always used `inherit`, without explicit reasons given. Most font properties should inherit but not `background`, `width`, `overflow`, `transition` as this is not their default value.

Comment (SassNinja) [reply scss/components/_off-canvas.scss]: @ncoden the in-canvas is supposed to just revert the off-canvas styles. Seems I was overusing the `inherit` a bit ;)
You're replacements of the properties in talk look good to me 👍

**Review by SassNinja — APPROVED** [review]

**Comment by SassNinja** [comment]
lgtm 👍 

The `inherit` was simply overused. Looks better now!

**Pull request merged**